src/Symfony/Component/HttpKernel/Exception/HttpException.php

```diff
@@ -21,6 +21,15 @@ class HttpException extends \RuntimeException implements HttpExceptionInterface
     private $statusCode;
     private $headers;
 
+    /**
+     * HttpException constructor.
+     *
+     * @param int             $statusCode An HTTP response status code
+     * @param string|null     $message    The internal exception message
+     * @param \Exception|null $previous   The previous exception
+     * @param array           $headers
+     * @param int|null        $code       The internal exception code
```

Comment:
The new description is nice ... but it's still not super clear to me. What's its purpose or why/when should I care about this?